wordpress.org/public_html/wp-content/plugins/plugin-directory/email/class-plugin-approved.php

```diff
@@ -29,32 +29,35 @@ function body() {
 To answer some common questions:
 
 * You must use SVN to upload your code -- we are unable to do that for you
-* Your SVN username is %2$s and your password is the same as you use to log in to WordPress.org
-* Your username is case sensitive
+* Your SVN username is `%2$s` and is case sensitive.
+* Generate your SVN password in your [WordPress.org profile](https://profiles.wordpress.org/%4$s/profile/edit/group/3/?screen=svn-password).
```

Comment:
Apologies, one more...

Since this is a URL that might change, and this is a translatable string, perhaps this should be moved outside of the text.

ie. `WordPress.org profile](%4$s)`
&
`"https://profiles.wordpress.org/{$this->user->user_nicename}/profile/edit/group/3/?screen=svn-password "`
&
`/* translators: 1: plugin name, 2: plugin author's username, 3: plugin slug, 4: link to plugin authors profile */`

Reply:
Is this what you had in mind? [7a0672d](https://github.com/WordPress/wordpress.org/pull/362/commits/7a0672dcab9e4190b5359c97b66d3b4808345350)

Reply:
Yes, but the `translators` comment needs to be before the `__()` string, not where it's used

Reply:
Ha. What am I doin'... 😏 😭